Automattic/wp-calypso
# Direct Link to Checkout from Site-less Jetpack Pricing Page

#### Changes proposed in this Pull Request

* When the Jetpack.com simple checkout flow is active ( feature flag `jetpack/siteless-checkout` ) send users directly to checkout
* ( since the flag is being set to false in all configs these changes will not be live on **merge** )

#### Testing instructions

1. Set `jetpack/siteless-checkout` to true in `jetpack-cloud-development.json`
2. Boot branch and navigate to `jetpack.cloud.localhost:3000/pricing`
3. Check the link of all the buttons for paid plans on the page
4. Verify they match the form of `https://wordpress.com/checkout/jetpack/[product_slug]` ( ⚠️ the link will not work yet, this PR is only about making sure it exists on this page ⚠️ )
5. Revert changes to `jetpack-cloud-development.json`, reboot branch, and redo steps 3 & 4, this time making sure the addresses match production ( `cloud.jetpack.com/pricing` )

related: 1200479326344990-as-1200507510970523
